extensions/CocoStudio/Armature/utils/CCArmatureDataManager.h

```diff
@@ -52,6 +52,7 @@ class  CC_EX_DLL CCArmatureDataManager : public CCObject
     static CCArmatureDataManager *sharedArmatureDataManager();
 
     static void purge();
+	static void destroyInstance();
```

Comment:
but studio is using `getInstance` & `purge` before, so I think it's better to deprecate `purge` and keep `destroyInstance`